extrafiles/sphinx-style/_templates/navbar.html

```diff
@@ -46,9 +46,11 @@
               <li class="hidden-sm">{% include "sourcelink.html" %}</li>
             {% endif %}
           </ul>
+          <!--
           {% block navbarsearch %}
             {% include "navbarsearchbox.html" %}
           {% endblock %}
+          -->
```

Comment:
Should we remove the search bar instead of commenting it?

Reply:
Agreed.